data/blog/wolvctf-2023/wannaflag.mdx

```diff
@@ -0,0 +1,335 @@
+---
+title: WolvCTF 2023 – WannaFlag Series
+date: '2023-03-20'
+draft: false
+authors: ['enscribe']
+tags: ['WolvCTF 2023', 'OSINT']
+summary: 'Hunting down a flag-stealing ransomware group—from Ethereum breadcrumbs to TF2 maps.'
+canonical: https://enscribe.dev/ctfs/wolv23/osint/wannaflag/
+---
+
+![WannaFlag Banner](/static/images/wolvctf-2023/banner.svg)
+
+<TOCInline toc={props.toc} asDisclosure open={false} exclude={'Intro'} />
+
+### Intro
+
+Over the last weekend, I played in WolvSec’s second CTF iteration with [Project Sekai](https://sekai.team/)—[WolvCTF 2023](https://ctftime.org/event/1866). We placed first in the open division, and throughout the solving process I became intrigued by a specific series of challenges placed under the OSINT category: **WannaFlag**. Telling a story of a supposed ransomware group which had been terrorizing the CTF community for the past several months, these series of challenges offered an opportunity for players to track down this group’s means of operation. Ultimately, the goal was to find WannaFlag’s kingpin through all possible methods.
+
+Project Sekai was the first to blood the entire series. Here was our thought process, notes, and conclusions.
+
+---
+
+## WannaFlag I: An Introduction
+
+> Welcome to WolvCTF’s OSINT Category! We have a bunch of great OSINT lined up, assuming nothing goes wrong hahahhahahhahah but why would it?  
+> For this challenge, find where the image was taken, and look at the Google Maps reviews!  
+> **Note**: Flags can be found in standard format `wctf{...}` for ALL OSINT challenges  
+> **Attachments**: [image.png](/static/images/wolvctf-2023/image.png)
+
+We’re first given a little bit of a warmup: find the location of the following object, and to view its Google Reviews:
+
+![image.png](/static/images/wolvctf-2023/image.png)
```

Comment:
Add alt text to all images used, highlight the important part(s) intended to convey by them. Same below.

Reply:
Resolved with [`480edeb` (#35)](https://github.com/project-sekai-ctf/sekai.team/pull/35/commits/480edeba1179ed1bb3f36b81aff1b051d4611c3c).